services/logging-operator/4.2.3/logging-operator-logging.yaml

```diff
@@ -14,7 +14,7 @@ spec:
         kind: HelmRepository
         name: kube-logging.github.io-charts
         namespace: kommander-flux
-      version: 4.1.0
+      version: 4.2.2
```

Comment:
https://github.com/kube-logging/helm-charts/releases
<img width="907" alt="image" src="https://github.com/mesosphere/kommander-applications/assets/5897740/b7ca2e63-2ae1-432e-bc84-528467cbd010">

very confusing.. but I guess logging-operator and logging-operator-logging charts are not necessarily in sync. logging-operator's latest version if 4.2.3, but logging-operator-logging is 4.2.2, but they are now not aligned here.